Lets Do This/Views/Campaign/LDTCampaignDetailActionButtonCell.m

```diff
@@ -23,6 +23,20 @@ - (void)awakeFromNib {
     [self.actionButton enable:YES];
 }
 
+- (UICollectionViewLayoutAttributes *)preferredLayoutAttributesFittingAttributes:(UICollectionViewLayoutAttributes *)layoutAttributes {
+    UICollectionViewLayoutAttributes *attributes = [[super preferredLayoutAttributesFittingAttributes:layoutAttributes] copy];
+
+    [self setNeedsLayout];
+    [self layoutIfNeeded];
+
+    CGRect newFrame = attributes.frame;
+    newFrame.size.width = CGRectGetWidth([UIScreen mainScreen].bounds);
+    newFrame.size.height = [self.contentView systemLayoutSizeFittingSize:UILayoutFittingCompressedSize].height;
+    attributes.frame = newFrame;
+    return attributes;
```

Comment:
Without this method, the action button sits in the bottom of the CampaignCell, horizontally centered and not stretching all the way to the left/right sides